### DISCUSSION TITLE
No more update?

### DISCUSSION THREAD

**Discussion by Tsaitou7361** [Tichau/FileConverter#322] [General]
Is there no more update with this program?
1.2.3 was publish in 2017, but now is 2023

**Comment by SsuJojo** [comment]
开发者好像走了

**Reply by Tsaitou7361** [reply]
oh no...

**Reply by Tsaitou7361** [reply]
話說你怎麼知道我說中文

**Reply by TheDarkXian** [reply]
怎么完善这个项目啊

**Reply by SsuJojo** [reply]
> 話說你怎麼知道我說中文

因为懒得翻译，而且不管你是什么语音肯定会用工具翻译(•́⌄•́๑)૭✧

**Reply by Tsaitou7361** [reply]
~~我以為會有人去看我的主頁~~

**Reply by Tsaitou7361** [reply]
> 如何完善這個項目啊

增加可轉換的檔案類型，改善多語言的相容性，修復issues等

**Comment by NeKoOuO** [comment]
> Is there no more update with this program?
> 1.2.3 was publish in 2017, but now is 2023

可是我怎麼看開發者好像還有在提交🤔

**Reply by SsuJojo** [reply]
我靠真的诶

**Comment by jie65535** [comment]
惊了，2.0来了

**Reply by TheDarkXian** [reply]
哪里

**Reply by jie65535** [reply]
https://github.com/Tichau/FileConverter/releases/tag/v2.0

**Reply by TheDarkXian** [reply]
好耶

**Reply by NeKoOuO** [reply]
七年了啊

**Comment by NeKoOuO** [comment]
> 惊了，2.0来了

詐屍了哈哈哈

**Comment by Tichau** [comment]
I close this thread since a new version has been deployed, don't hesitate to open it again if the project does not see any updates.
